src/quo/components/drawers/bottom_actions/view.cljs

```diff
@@ -7,43 +7,53 @@
     [react-native.core :as rn]))
 
 (def default-props
-  {:button-one-type :primary
-   :button-two-type :grey})
+  {:button-one-type     :primary
+   :button-two-type     :grey
+   :customization-color :blue})
 
 (defn- view-internal
   "Options:
 
-  :actions - keyword (default nil) - :1-action/:2-actions
-  :description - string (default nil) - Description to display below the title
-  :button-one-label - string (default nil) - Label for the first button
-  :button-two-label - string (default nil) - Label for the second button
-  :button-one-press - fn (default nil) - Function to call when the first button is pressed
-  :button-two-press - fn (default nil) - Function to call when the second button is pressed
-  :theme - :light/:dark
-  :scroll - bool (default false) - Whether the iOS Home Indicator should be rendered
-  :button-one-type - same as button/button :type
-  :button-two-type - same as button/button :type"
+       :actions - keyword (default nil) - :1-action/:2-actions
+       :description - string (default nil) - Description to display below the title
+       :button-one-disabled? - bool (default false) - Whether the first button is
```

Comment:
I don't think the naming is clear enough here. Maybe `create` and `continue` would be better

Reply:
imo it should be `button-one-props` and `button-two-props` and we pass the map along 👍

Reply:
@J-Son89 how should i change the doc to reflect refactoring to maps?